### DISCUSSION TITLE
Como puedo obtener la cantidad de días de un campo de fecha para un campo numérico

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#379] [2. Questions and Answers]
Hola

Tengo dos campos, 1 de fecha y 1 de número:  Cómo puedo obtener los n° de días que tengo desde la fecha que ingreso hasta hoy?.
Se que es question exec de tipo preload.

Gracias =>

**Comment by acedward** [comment]
Te sirve en el validate?
ahi puedes leer el context responses#self
ahi viene en en el primer argumento del arreglo la fecha en unix time, y la puedes restar contra new Date().getTime() y tener la cantidad de dias 
finalmente con SET_RESPONSES puedes escribir en otro campo

**Reply by NicolasPLG** [reply]
De hecho era validate y no preload (mal ngreso a mi consulta :/), buscaré algunos ejemplos para la resta de estos argumentos. Gracias @acedward 
Cerraré esta discusión copiando el resultado del question exec

**Reply by NicolasPLG** [reply]
hola @acedward salió esto

```
    "exec" : {
        "validate" : {
            "src" : "function run(){
const commands = [];
const initTime = new Date(context['responses#fecha_inicio_farmaco_sesion']).getTime();
const endTime = new Date().getTime();
const tiempo = (endTime - initTime);
const dias = (tiempo /(1000 * 3600 * 24));
commands.push({cmd: 'SET_RESPONSES',value: dias});
commands.push({cmd: 'SET_READONLY',value: 'true'});
console.log(commands);
return commands;}", 
            "context" : "responses#self,responses#fecha_inicio_farmaco_sesion"
        }
    }
```
Lo revisamos con @SeIturra y @alelo y no se está ejecutando el exec (revisamos colocando un valor fijo en el SET_RESPONSES).

Gracias =>
